test/testcafe/framework/page-objects/FactorEnrollPasswordPageObject.js

```diff
@@ -2,7 +2,16 @@ import BasePageObject from './BasePageObject';
 
 const passwordFieldName = 'credentials\\.passcode';
 const confirmPasswordFieldName = 'confirmPassword';
-
+const requirementsSelector = '.password-authenticator--rules';
```

Comment:
Can we use `data-se` attribute here to fetch the elements and not rely on class names?

Reply:
Will address it when I work on admin initiated password reset / password will expire soon.

Reply:
Fixed in https://github.com/okta/okta-signin-widget/pull/1281